src/components/promotion/PromotionDetails.tsx

```diff
@@ -1,42 +1,181 @@
-import { HeartOutlined } from '@ant-design/icons';
+import './PromotionDetails.css';
+
+import { ClockCircleOutlined, HeartOutlined } from '@ant-design/icons';
+import { Col, Row, Typography } from 'antd';
 import React, { CSSProperties, ReactElement } from 'react';
 
-import { Promotion } from '../../types/promotion';
+import { Promotion, Schedule } from '../../types/promotion';
+
+const { Title, Text } = Typography;
 
 const styles: { [identifier: string]: CSSProperties } = {
-  descriptionContainer: {
-    padding: 10,
+  detailsContainer: {
+    paddingLeft: 10,
     textAlign: 'left',
     width: '100%',
   },
+  footer: {
+    color: '#8B8888',
+    fontSize: '0.8em',
+  },
   header: {
-    display: 'flex',
-    justifyContent: 'space-between',
+    fontFamily: 'ABeeZee',
+    fontSize: '1em',
+    lineHeight: '1em',
+    paddingBottom: 5,
   },
   heart: {
-    marginTop: 7,
-    textAlign: 'right',
+    fontSize: '1.5em',
+  },
+  promotionName: {
+    fontSize: '1.5em',
+    fontWeight: 'normal',
+    marginBottom: 0,
+  },
+  restaurantName: {
+    fontSize: '0.9em',
+    lineHeight: '0.9em',
+    textDecoration: 'underline',
+  },
+  schedule: {
+    color: '#8B8888',
+    fontSize: '0.9em',
+  },
+  scheduleContainer: {
+    paddingTop: 10,
+    paddingBottom: 10,
   },
 };
 
 export default function PromotionDetails({
   name,
+  dateAdded,
   description,
   expirationDate,
-  restaurantName = 'Restaurant',
+  restaurantName,
+  schedules,
 }: Promotion): ReactElement {
+  /**
+   * Returns display text for age of promotion.
+   *
+   * - If dateAdded < 1 day, displays "X hours ago"
+   * - If 1 day <= dateAdded < 1 week, displays "X days ago"
+   * - If 1 week <= dateAdded < 1 month, displays "X weeks ago"
+   * - If 1 month <= dateAdded < 1 year, displays "X months ago"
+   * - If 1 year <= dateAdded, displays "X years ago"
+   */
+  const promotionAge = (dateAdded: string) => {
```

Comment:
I wonder if we can leverage an NPM package for this, maybe something like https://www.npmjs.com/package/time-ago? 

Regardless, this is really cool

Reply:
Oooh good idea, what do you think of https://date-fns.org/? It seems like it has a lot of support and we could potentially use their other functions if we have needs for it in the future

Usage ex:
```
formatDistance(subDays(new Date(), 3), new Date())
```

Reply:
Oooh that looks even better, let's go w that then!